### ISSUE TITLE
[BUG] TimeSlider Keyboard and tooltip

### ISSUE THREAD

**Issue by jolevesq** [Canadian-Geospatial-Platform/geoview#2505]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

Time slider slider is not keyboard accessible

Tooltip on layer list is like: for 1800-1900, the numbers shown are 1800-01-01T14:22:55 – 1900-05-29T23:19:25
### Expected Behavior

We should be able to use keyboard on selected handle of time slider to move forward/backward. To make it easier, we should try to see if we can increment/decrement the step

We should keep the interval of slider to define the tooltip accuracy to apply to date

In our ui we have have format use for the date: (YYYY-MM-DD) or YYYY, ....

### Steps To Reproduce

1. https://canadian-geospatial-platform.github.io/geoview/public/demos-navigator.html?config=./configs/navigator/10-package-time-slider.json
2. Tag to time slider
3. No keyboard controls
